functional_plant_indicators_seminat.Rmd

```diff
@@ -49,33 +49,33 @@ knitr::kable(metaData)
 
-- as test data we use plant community data from (1) the ANO monitoring scheme (cf. Tingstad et al. 2019, contains semi-natural grasslands(T32), semi-natural wet meadows (V10), heathlands (T34), and we also decided to include near-semi-natural meadows (T41) ) and (2) from the ASO monitoring scheme (cf. Bär et al. 2021, targets semi-natural grasslands (T32) only)
+- as test data we use plant community data from (1) the ANO monitoring scheme (cf. Tingstad et al. 2019, contains semi-natural grasslands T32, semi-natural wet meadows V10, heathlands T34, and we also decided to include near-semi-natural meadows T41) and (2) from the ASO monitoring scheme (cf. Bär et al. 2021, targets semi-natural grasslands T32 only)
 - as reference data we use (3) generalized species lists from the Norwegian categorization system for eco-diversity (NiN) (cf. Halvorsen et al. 2020)
-- (4) Swedish plant indicator data for light, moisture, pH, and nitrogen from Tyler et al. (2021)
+- finally, (4) Swedish plant indicator data for light, moisture, pH, and nitrogen from Tyler et al. (2021)
 
 (1) ANO monitoring data:
-ANO stands for 'areal-representativ naturovervåking', i.e. 'area representative nature monitoring'. 1000 sites are randomly distributed across mainland Norway and visited within a 5-year cycle. Each ANO site spans a 500 x 500 m grid cell, and the data collection at each ANO site includes up to 12 evenly distributed vegetation analyses in 1 x 1 m plots (up to 12, because some of these evenly distributed points may be in water or otherwise inaccessible). For the vegetation analyses, the cover of each vascular plant species in the plot is recorded. Every vegetation analysis is accompanied by an assessment of the ecosystem the plot lies in, including ecosystem type and some additional variables related to ecosystem-specific drivers of state. In the analysis in this document, we only use the plots which were classified as one of the semi-natural ecosystem types in the Norwegian categorization system for eco-diversity (NiN).
+ANO stands for 'areal-representativ naturovervåking', i.e. 'area representative nature monitoring'. 1000 sites are randomly distributed across mainland Norway and visited within a 5-year cycle. Each ANO site spans a 500 x 500 m grid cell, and the data collection at each ANO site includes up to 12 [?] evenly distributed vegetation analyses in 1 x 1 m plots (up to 12, because some of these evenly distributed points may be in water or otherwise inaccessible). For the vegetation analyses, the cover of each vascular plant species in the plot is recorded. Every vegetation analysis is accompanied by an assessment of the ecosystem the plot lies in, including ecosystem type and some additional variables related to ecosystem-specific drivers of state. In the analysis in this document, we only use the plots which were classified as one of the semi-natural ecosystem types in the Norwegian categorization system for eco-diversity (NiN).
```

Comment:
[?] - there are 18 plots per ANO site, arent there? (except in 2019, when there were 12?)

Reply:
true, corrected for all files in the upcoming pull request. Thanks for spotting :)